pkg/datapath/config.go

```diff
@@ -92,6 +92,10 @@ type CompileTimeConfiguration interface {
 
 	// IsHost returns true if the endpoint is the host endpoint.
 	IsHost() bool
+
+	// DisableSipVerification returns true if the endpoint wishes to skip
+	// source IP verification
+	DisableSipVerification() bool
```

Comment:
This field is not autogenerated, so could be `DisableSIPVerification() bool`

Reply:
fixed capitalization, thanks